### PR TITLE
Update github id

### PR DESCRIPTION
**Pull request by yuting0203**
## CHECKS :warning:

Please make sure you are aware of the following.

- [ ] **The changes in this PR doesn't have private information

### CODE REVIEW

**Review by mikan373 — APPROVED** [review]
I approve the change you made

**Review by tsoibet — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Build@Mercari Training Program 2022
```
Comment (Riya-code) [inline README.md:1]: LGMT

**Review by Riya-code — COMMENTED** [review]
Reviewed

**Pull request merged**